examples/low-level-api/listOffers.ts

```diff
@@ -12,22 +12,22 @@ const { asyncWith, CancellationToken } = utils;
 const cancellationToken = new CancellationToken();
 
 async function list_offers(conf: rest.Configuration, subnetTag: string): Promise<void> {
-  let client = await conf.market();
-  let market_api = new Market(client);
-  let dbuild = new yp.DemandBuilder();
+  const client = await conf.market();
+  const market_api = new Market(client);
+  const dbuild = new yp.DemandBuilder();
```

Comment:
okay, here's where my js is not sharp enough -> doesn't `const` suggest that the variable will not be modified? I guess the meaning is obvious when it comes to immutable values... but here it's a mutable and while indeed, we won't be assigning to the `dbuild`, I guess its value _will_ get modified, won't it? ... 

that doesn't feel like a constant to me then... but if it's still okay by the js lingo, I'm okay

Reply:
yeah .. JS can be weird sometimes. In JS, `const` means that the identifier can’t be assigned, not that values is immutable. And linter show errors if you use `let` for identifiers which are not reassigned.